http_service/bugbug_http/app.py

```diff
@@ -190,6 +190,43 @@ def schedule_job(
     )
 
 
+def prepare_queue_job(job: JobInfo, job_id: Optional[str] = None) -> Queue:
+    """Prepare queue data for for enqueueing multiple jobs"""
+    job_id = job_id or get_job_id()
+
+    redis_conn.mset({job.mapping_key: job_id})
+    return Queue.prepare_data(
+        job.func,
+        *job.args,
+        job_id=job_id,
+        ttl=QUEUE_TIMEOUT,
```

Comment:
In `schedule_job`, we are also passing `job_timeout`. Isn't it available here?

Reply:
@marco-c 
On passing `timeout=BUGZILLA_JOB_TIMEOUT`, the test was failing with error: `TypeError: prepare_data() got multiple values for argument 'timeout'`

```
    return Queue.prepare_data(
        job.func,
        *job.args,
        job_id=job_id,
        timeout=BUGZILLA_JOB_TIMEOUT,
        ttl=QUEUE_TIMEOUT,
        failure_ttl=FAILURE_TTL,
    )
```

**Error Log**:
https://community-tc.services.mozilla.com/tasks/eXtoGgbxSTufCkFhJzRA1w/runs/0/logs/public/logs/live.log

Reply:
We should figure it out as we really want a different timeout for these jobs.

I suggest adding some logging to figure out why it is failing, and/or look at the difference between `prepare_data` (https://github.com/rq/rq/blob/6692587f74261a2121408f6e231d0ab46630ecc0/rq/queue.py#L409) and `enqueue` (https://github.com/rq/rq/blob/6692587f74261a2121408f6e231d0ab46630ecc0/rq/queue.py#L500) arguments.

Reply:
Sure, I will look into this.

Reply:
@marco-c Have done the required changes. Please review.